projects/packages/image-cdn/src/compatibility/photon.php

```diff
@@ -9,6 +9,10 @@
 
 namespace Automattic\Jetpack\Image_CDN\Compatibility;
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( 0 );
```

Comment:
On a lighter note, it would be fun to exit with a non-zero code to break the apps for script kiddies. 😄